joschahenningsen/TUM-Live
# Remove OCR

### Motivation and Context
Since whisper provides better (i.e. cleaner) data for lecture searches, OCR isn't needed anymore. 

### Description
Remove all OCR related code.

### Steps for Testing
1. Build Worker Docker file
2. Check course for console errors
3. Upload VOD without errors
